src/simpleroi.cpp

```diff
@@ -0,0 +1,119 @@
+// simpleroi.cpp
+#include "simpleroi.h"
+#include "chainparams.h"
+#include "masternodeman.h"
+#include <inttypes.h>
+#include <univalue.h>
+#include "validation.h"
+
+const int CSimpRoiArgs::nStakeRoiHrs = 3;	// 3 hour averaging for smooth stake values
+
+int64_t CSimpleRoi::getTimeDiff(CSimpRoiArgs& csra, uint64_t n_blocks, int nHeight)
+{
+    if (nHeight < n_blocks) n_blocks = nHeight;
+    csra.pb0 = chainActive[nHeight - n_blocks];
+// return timeDiff
+    return csra.pb->GetBlockTime() - csra.pb0->GetBlockTime();
+}
+
+// returns count enabled or zero on error
```

Comment:
```suggestion
// returns count enabled or zero on error
```
This says it will return zero on error, but later it returns negative numbers on error

Reply:
see line 76